Description: Always use Debian's libmpeg2 library
 Upstream fs-uae ships with its own version of libmpeg2. In Debian,
 it's always preferred to avoid embedded libraries as these pose a
 potential security risk. Moreover, in this particular case, the
 embedded version of libmpeg2 results in an FTBFS on alpha, armel
 and powerpc. Both issues can be resolved by forcing fs-uae to use
 Debian's own version of libmpeg2 which can be used by build-depending
 on libmpeg2-4-dev. This patch is necessary because upstream fs-uae
 currently does not make the use of an external version of libmpeg2
 configurable
 .
--- fs-uae-2.6.1+dfsg.orig/Makefile.am
+++ fs-uae-2.6.1+dfsg/Makefile.am
@@ -1,6 +1,6 @@
 # Makefile.am for FS-UAE
 
-SUBDIRS = libmpeg2
+# SUBDIRS = libmpeg2
 
 WARNINGS =
 
--- fs-uae-2.6.1+dfsg.orig/configure.ac
+++ fs-uae-2.6.1+dfsg/configure.ac
@@ -112,8 +112,8 @@ AM_CONDITIONAL([BUILTIN_GLEE], [false])
 PKG_CHECK_MODULES([GLEW], [glew])
 
 # Todo: make configurable
-# PKG_CHECK_MODULES([LIBMPEG2], [libmpeg2 libmpeg2convert])
-AM_CONDITIONAL([BUILTIN_LIBMPEG2], [true])
+PKG_CHECK_MODULES([LIBMPEG2], [libmpeg2 libmpeg2convert])
+AM_CONDITIONAL([BUILTIN_LIBMPEG2], [false])
 
 AM_CONDITIONAL([BUILTIN_MANYMOUSE], [true])
 
@@ -627,7 +627,7 @@ OPT_FEATURE([FS_EMU_DRIVERS], [fs_emu_dr
 
 AC_DEFINE([FPU_UAE], [1], [Define to 1])
 
-AC_CONFIG_SUBDIRS([libmpeg2])
+# AC_CONFIG_SUBDIRS([libmpeg2])
 
 # Write Makefile
 
